importer/importer.py

```diff
@@ -43,8 +43,17 @@ def __init__(self, countryname, languagename):
         self.file_content = self.load_mapping_file(countryname, languagename)
 
 
-def make_query(specific_table):
-    return ('select DISTINCT *  from `{}`').format(specific_table)
+def make_query(specific_table, offset):
+    """
+    Generate a query to retrieve data from database.
+
+    :param specific_table: Name of table to retrieve data from.
+    :param offset: Optional offset to start from.
+    """
+    query = 'select DISTINCT *  from `{}`'.format(specific_table)
+    if isinstance(offset, int):
+        query = query + " LIMIT {}, 18446744073709551615;".format(str(offset))
```

Comment:
`query = query + " LIMIT..."` can be replaced by `query += " LIMIT..."`

18446744073709551615:
If this is just an arbitrary large number then I would make it a variable called something like that to clarify.